.changeset/hungry-mangos-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+"@journeyapps/wa-sqlite": patch
```

Comment:
Could we make this a `minor` change? Maybe that could help a little bit to avoid people mistakingly downgrading the package.